src/utils/router_state.rs

```diff
@@ -260,15 +262,116 @@ pub fn is_vertiport_available(
             }
         })
         .count();
+    let res = if num_vertipads > 1 {
+        let vehicles_at_vertiport =
+            get_all_vehicles_scheduled_for_vertiport(&vertiport_id, date_to, existing_flight_plans);
```

Comment:
If vertiport has more than 1 vertipad, we use different - more complicated check, when there is less than 1, it's an old way of checking - I assume there is 1 vertipad (never 0), otherwise vertiport would have no availability.